src/rucksack.cr

```diff
@@ -2,17 +2,23 @@ require "openssl"
 
 class Rucksack
   class NotFound < Exception; end
+
```

Comment:
Not a fan of these blank lines, but if that's what crystal format
wants then, oh well, probably easier to keep them than to argue
with it every time... ;)